lib/Sema/CSDiag.cpp

```diff
@@ -2490,6 +2490,17 @@ diagnoseUnviableLookupResults(MemberLookupResult &result, Type baseObjTy,
           diagnose(loc, diag::did_you_mean_raw_type);
           return; // Always prefer this over typo corrections.
         }
+      } else if (baseObjTy->isAny()) {
+        if (auto DRE = dyn_cast<DeclRefExpr>(baseExpr)) {
+          auto NameStr = DRE->getDecl()->getName().str();
```

Comment:
I copied this from another diagnostic from another file. I'll take a closer look at some of the other diagnostics and update when I open the new PR.